src/probe/probe.c

```diff
@@ -906,6 +906,8 @@ static void probe_cb_free(void *arg, enum notify_id type, void *data)
 		tr_err(&pr_tr, "probe_cb_free(): probe_point_remove() failed");
 }
 
+extern struct ipc_comp_dev logging_comp;
+
```

Comment:
This is ugly and not sure if we want to keep this. I'll made one try to get rid of this, but it broke the notifier logic. I'll retry again. Alternative we could drop have more explicit callbacks between the backend and probe client code, and not rely on comp_buffer struct and the notifier.h interface.

Reply:
V3 removed this usage of ipc_comp_dev.